internal/venice-common/src/main/java/com/linkedin/venice/utils/RegionUtils.java

```diff
@@ -59,4 +59,14 @@ public static Set<String> parseRegionsFilterList(String regionsFilterList) {
     }
     return Stream.of(regionsFilterList.trim().split(REGION_FILTER_LIST_SEPARATOR)).collect(Collectors.toSet());
   }
+
+  /**
+   * A helper function to compose a region list with {@link #REGION_FILTER_LIST_SEPARATOR}.
+   * This is the reverse of {@link #parseRegionsFilterList(String)}.
+   * @param regions, a set of regions
+   * @return a string of regions separated by {@link #REGION_FILTER_LIST_SEPARATOR}
+   */
+  public static String composeRegionList(Set<String> regions) {
```

Comment:
does it deserve a method for just string join?

Reply:
I think it's okay to have it as a method. When we change how to parse the region filter, people only need to update `composeRegionList` once if we end up using it at many places.